server.js

```diff
@@ -117,6 +117,7 @@ async function storeCard(req, res) {
       // remove 64-bit value from response
```

Comment:
lets update the comment to explain this better ... it is not possible to serialize a 64 bit value (or bigInt) into a JSON object which is what micro does with its send [method](https://www.npmjs.com/package/micro)

Reply:
Sounds good! I can update that!